tools/json2github.py

```diff
@@ -0,0 +1,666 @@
+
+You can install this like this:
+
+    $ python3 -m venv venv
+    $ source venv/bin/activate
+    $ pip install PyGithub
+
+To use it, you first need to obtain a bugs and milestone json file from lp2gh:
+
+    https://github.com/Swiftb0y/lp2gh/blob/mixxx-import-fork/docs/moving_issues.rst#exporting-your-bugs
+    https://github.com/Swiftb0y/lp2gh/blob/mixxx-import-fork/docs/moving_issues.rst#exporting-your-milestones
+
+Its important you are using my Fork with the `mixxx-import-fork` branch as I
+made some slight modifications to add features.
+
+You can use it like this:
+
+    $ python json2github.py --repo Holzhaus/mixxx-gh-issue-migration \
+      --token mytoken --output-file=mixxx_bugs.json \
+      mixxx_bugs.json mixxx_milestones.json
+
+"""
+import argparse
+import binascii
+import datetime
+import json
+import logging
+import random
+import re
+import textwrap
+import time
+import github
+
+LAUNCHPAD_STATUS_MAP = {
+    "Confirmed": ["confirmed"],
+    "Fix Committed": [],
+    "Fix Released": [],
+    "Incomplete": ["incomplete"],
+    "In Progress": [],
+    "Invalid": ["invalid"],
+    "New": [],
+    "Triaged": ["confirmed"],
+    "Won't Fix": ["wontfix"],
+    "Opinion": ["opinion"],
+    "Expired": ["expired"],
+}
+
+LAUNCHPAD_IMPORTANCE_MAP = {
+    "Critical": ["party stopper", "bug"],
+    "High": ["bug"],
+    "Low": ["bug"],
+    "Medium": ["bug"],
+    "Undecided": [],
+    "Wishlist": ["feature"],
+}
+
+LAUNCHPAD_USER_MAP = {
+    "aart": "amvanbaren",
+    "alex-jercaianu": "jercaianu",
+    "alex.barker": "kwhat",
+    # (not sure, email on launchpad matches
+    # with domain linked on github profile)
+    "another-rob": "borfo",
+    "arbeit-u": "dg3nec",
+    "arli0715": "dodler",
+    "badescunicu": "badescunicu",
+    "be.ing": "Be-ing",
+    "bencoder": "bencoder",
+    "bkgood": "bkgood",
+    "bruno-buccolo": "buccolo",
+    "buzz-dee": "BuZZ-dEE",
+    "cardinot": "cardinot",
+    "chloe-avrillon": "DJChloe",
+    "crislacerda": "crislacerda",
+    "daniel-64studio": "danielhjames",
+    "daschuer": "daschuer",
+    "default-kramer": "default-kramer",
+    "dj-kaza": "kazakore",
+    "eradkoff12": "radkoff",
+    "ewanuno": "ewanuno",  # (not sure)
+    "federicobriata": "federicobriata",
+    "ferranpujol": "ferranpujolcamins",
+    "foss4": "foss-",
+    "frank-breitling": "fkbreitl",
+    "gamegod": "asantoni",
+    "gmsoft": "gmsoft-tuxicoman",
+    "goddisignz": "goddisignz",
+    "hacksdump": "hacksdump",
+    "hile": "hile",
+    "holthuis-jan": "Holzhaus",
+    "htown202": "htown101",
+    "iamcodemaker": "iamcodemaker",
+    "ironstorm-gmail": "deftdawg",
+    "jatwill": "idcmp",
+    "jean-claveau-g": "jclaveau",
+    "jessboerner": "doodlebro",
+    "jmigual": "jmigual",
+    "joerg-ubuntu": "JoergAtGithub",
+    "johan-lasperas": "johanLsp",
+    "josepma": "JosepMaJAZ",
+    "juha-pitkanen": "JuhaPit",
+    "jus": "esbrandt",  # codespell:ignore jus
```

Comment:
I line comments don't work, you need to add `jus` to the `.codestellignore` file.

Reply:
Yeah I'm aware. There is a PR that introduces inline ignores. I didn't want to add "jus" to the global `.codespellignore` because it could be a likely typo. I'd just wait for https://github.com/codespell-project/codespell/pull/2400 before merging. :shrug:

Reply:
The alternative to waiting is to add the entire line to a `.codespellignorelines` file and adding `--exclude-file=.codespellignorelines` to the codespell command line. Then you don't have to worry about false negatives.